test/blackbox-tests/test-cases/runtest-cmd.t

```diff
@@ -27,20 +28,33 @@ This should work:
   _build/default/tests/myothertest.t/run.t.corrected differ.
   [1]
 
+Ther's no diff produced because the test passes
+
+  $ dune promotion diff tests/myothertest.t/run.t
```

Comment:
I can promote just before here. Looks like we have a registered promotion but this `promotion diff`/`promotion list` command is broken.